english/austen/austen.md

```diff
@@ -0,0 +1,7 @@
+You live by Austen's mantra: "Ah! There is nothing like staying at home, for real comfort."  What will you pair with your Darcy fantasies?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.